src/docs/development/testing.mdx

```diff
@@ -114,39 +110,37 @@ End-to-end tests rely on the ability to find elements on the screen and interact
 -   **xPath Selectors**: find selector within xPath (e.g., `//*[@id="@@pxb-drawerlayout-content"]/div/header/div/button`)
 -   **ID Selectors**: find attribute id within elements (e.g., `#login-button`)
 
-See selector strategies on [browserstack](https://www.browserstack.com/guide/css-selectors-in-selenium) and [software testing help](https://www.softwaretestinghelp.com/css-selector-selenium-locator-selenium-tutorial-6/)
+> You can read more about different selector strategies on [browserstack](https://www.browserstack.com/guide/css-selectors-in-selenium) and [software testing help](https://www.softwaretestinghelp.com/css-selector-selenium-locator-selenium-tutorial-6/).
 
 Each of these methods has their own advantages and disadvantages. A common pain point in E2E test automation is that modern Javascript platforms are constantly changing, particularly in the open source community. Many of these frameworks automatically (and dynamically) generate IDs and classes for components resulting in a constantly moving target for automation tooling. Identifying elements by these values (i.e., using IDs, CSS, or xPath locators _without_ property attributes) makes your tests brittle because they are subject to change any time a new version comes out (or even any time the page is reloaded).
 
 One way to combat this in your own applications is to use dedicated attributes for testing (e.g., `test-id` or `dev-id`). By adding this test-specific attribute to the element, identifying the correct underlying component should be safer. Because this attribute is test-specific and the application logic does not use it, it will be less likely to change throughout the product lifecycle. Combining this approach with other selectors is a common and reliable approach for element identification.
 
 ```html
-<button test-id="login-button" class="default-button">Login</button>
-```
+// Component code
 
-Selector to use in a test
+<button id="login-button" class="default-button">Login</button>
 
-```
-WebElement click = driver.findElement(By.id("login-button"));
+// Selector to use in a test WebElement click = driver.findElement(By.id("login-button"));
```

Comment:
looks like prettier may have messed this code block up

Reply:
think its better now